tests/unit/type/stream-cgroups.tcl

```diff
@@ -357,6 +357,106 @@ start_server {
         assert {[lindex $reply 0 1] eq {client1}}
     }
 
+    test {XAUTOCLAIM can claim PEL items from another consumer} {
+        # Add 3 items into the stream, and create a consumer group
+        r del mystream
+        set id1 [r XADD mystream * a 1]
+        set id2 [r XADD mystream * b 2]
+        set id3 [r XADD mystream * c 3]
+        r XGROUP CREATE mystream mygroup 0
+
+        # Client 1 reads item 1 from the stream without acknowledgements.
+        # Client 2 then claims pending item 1 from the PEL of client 1
+        set reply [r XREADGROUP GROUP mygroup client1 count 1 STREAMS mystream >]
+        assert_equal [llength [lindex $reply 0 1 0 1]] 2
+        assert_equal [lindex $reply 0 1 0 1] {a 1}
+        r debug sleep 0.2
```

Comment:
let's avoid debug sleep, if you need to let the wall clock move, but don't need to block other client connections to the server, use `after`